terraform/dms/dms.tf

```diff
@@ -32,7 +31,6 @@ resource "aws_dms_endpoint" "target" {
   ssl_mode                        = "require"
   secrets_manager_arn             = try(data.aws_secretsmanager_secret.target[each.key].arn, null)
   secrets_manager_access_role_arn = data.aws_iam_role.secrets_manager.arn
-  extra_connection_attributes     = try("secretsManagerEndpointOverride=${aws_vpc_endpoint.secrets_manager[0].dns_entry[0].dns_name}", null)
```

Comment:
I think these extra connection attributes are still needed, it's just the names will come from elsewhere. Is that right @dark5un?

Reply:
These are not needed after your vpc dns changes. This has been tested in dev.